src/platform/utilities/api/index.js

```diff
@@ -70,6 +71,7 @@ export function apiRequest(resource, optionalSettings = {}, success, error) {
     headers: {
       'X-Key-Inflection': 'camel',
       'Source-App-Name': window.appName,
+      'X-CSRF-Token': csrfTokenStored,
```

Comment:
What happens if `csrfTokenStored` is `null`?

Reply:
Validation is only enforced on non-HEAD and non-GET requests (e.g. POST, DELETE, PUT, PATCH). Since most likely the first request is usually `/maintenance_windows/` which is a GET, there is no problem there. Moreover, this call will set the `X-CSRF-Token` into the `localStorage`.

In addition, Logging violations to Sentry will hopefully let us see if this scenario might occur